VisualPinball.Engine/Common/Constants.cs

```diff
@@ -28,6 +28,17 @@ public static class Constants
 		public const float Gravity = 1.81751f;
 	}
 
+	/// <summary>
+	/// Layer of the table items.
+	/// Currently only one layer, but might be more in the future.
+	/// </summary>
+	public static class Layer

```

Comment:
Can you name it `LayerConstants`?

Reply:
Ok. Main point of this is that you change this:

public const int ChildObjectsLayer = 8;

to anything larger than 8 because 8 is used as PostProcessing Layer in HDRP by default.